contributing/projet-management.md

```diff
@@ -0,0 +1,105 @@
+# Project management guidelines
+
+|IMPORTANT: The project management guidelines are only applicable to AMP Stories at this stage.|
+| :--- |
+
+# Project boards
+
+In addition to [Milestones](https://github.com/ampproject/amp-wp/milestones), which are used to manage releases, project boards are used to manage issue statuses.
+
+### Definition project board
+
+The [Definition](https://github.com/ampproject/amp-wp/projects/14) project board covers the pipeline which issues go through in preparation for execution. The board contains the following columns:
+
+* Revisit Later
+* Prioritization
+* Acceptance Criteria
+* Implementation Brief / Estimate
+* Implementation Brief Review
+* Estimate
+
+### Execution project board
+
+The [Execution](https://github.com/ampproject/amp-wp/projects/15) project board covers the execution pipeline which issues go through for implementation. The board contains the following columns:
+
+* Blocked
+* Backlog
+* To Do
+* In Progress
+* Code Review
+* QA
+* Approval
+* Done
+
+## Labels
+
+The labels below are utilized to categorize issues:
+
+* Type: `{type}` = the issue type (ex. `Type: Bug`, `Type: Feature`, `Type: Support`)
+* P`{priority}` = the priority of the task (ex. `P1`, `P2`, `P3`)
+* Size: `{size`} = the issue size (ex. `S`, `M`, `L`)
+* Sprint: `{sprint_number`} = the sprint associated to the issue (ex. `Sprint: 1`, `Sprint: 2`, `Sprint: 3`)
+
+## Life of an issue
+
+|IMPORTANT: We use GitHub issues to track all task statuses, therefore PRs should **only** be associated with an issue, **not** assigned a label, project, and/or milestone.|
+| :--- |
+
+### Triage
+
+The [GitHub issues](https://github.com/ampproject/amp-wp/issues) view serves as the “Awaiting Triage” backlog.
+
+1. An issue is created.
+1. The issue “Type” label is assigned.
+
+### Issue cycle by type
+
+#### Type: `Bug`, `Enhancement`, `Feature`
+
+##### Definition
+
+| <br> Step | <br> Task | &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<br> Role  |
+| :--- | :--- | :--- |
+| 1. | An issue requiring work is added to the [Definition](https://github.com/ampproject/amp-wp/projects/14) project board (automatically added to Prioritization column). | `Product Manager` `Program Manager`
+| 2. | The issue is assigned a “Priority” and moved to the “Acceptance Criteria” column. | `Product Manager` `Program Manager`
+| 3. | “Acceptance Criteria” are added to the issue description, and the issue is moved to the “Implementation Brief” column. | `Product Manager` `Lead Engineer`
+| 4. | “Implementation Brief” is added to the issue description, and the issue is moved to the “Implementation Brief Review” column. NB: an estimate (step 6) may be added as part of this step. | `Engineer`
```

Comment:
I would recommend removing estimating from this step, as it is a bit confusing in the process.